intelmq/bots/parsers/alienvault/parser-otx.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import sys
+import json
+
+from intelmq.lib import utils
+from intelmq.lib.bot import Bot
+from intelmq.lib.harmonization import DateTime
+from intelmq.lib.message import Event
+
+HASHES = {
+    'FileHash-SHA256': 'SHA-256',
+    'FileHash-SHA1': 'SHA-1',
+    'FileHash-MD5': 'MD5'
+}
+
+
+class AlienVaultOTXParserBot(Bot):
+
+    def process(self):
+        report = self.receive_message()
+        if (report is None or not report.contains("raw") or
+                len(report.value("raw").strip()) == 0):
+            self.acknowledge_message()
+            return
+
+        time_observation = DateTime().generate_datetime_now()
+        raw_report = utils.base64_decode(report.value("raw"))
+
+        for pulse in json.loads(raw_report):
+            comment = "author: " + pulse['author_name'] + "; name: " + \
```

Comment:
using [string formatting](https://docs.python.org/2.7/library/functions.html?highlight=format#format) us usually more clear. Example:

                comment = ("author: {}; name: {}; description: {}"
                           "".format(pulse['author_name'], pulse['name'],
                                     pulse['description']))

The resulting string can be seen immediately.